extension.js

```diff
@@ -60,6 +60,7 @@ class MprisLabel extends PanelMenu.Button {
 		this.settings.connect('changed::extension-index',this._updateTrayPosition.bind(this));
 		this.settings.connect('changed::extension-place',this._updateTrayPosition.bind(this));
 		this.settings.connect('changed::show-icon',this._setIcon.bind(this));
+		this.settings.connect('changed::use-album',this._setIcon.bind(this));
```

Comment:
missing `this.settings.connect('changed::album-size',this._setIcon.bind(this));`
```suggestion
		this.settings.connect('changed::show-icon',this._setIcon.bind(this));
		this.settings.connect('changed::use-album',this._setIcon.bind(this));
		this.settings.connect('changed::album-size',this._setIcon.bind(this));
```